<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMiniVariable</class>
 <widget class="QWidget" name="WidgetMiniVariable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>150</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Variable Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leVarName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Current Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leValue"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTime"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Equation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEquation"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Depends On</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="conDepends"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
